_includes/configuration/available-context-menu-items.md

```diff
@@ -0,0 +1,20 @@
+## Available context menu items
+
+The following table shows all context menu items, including items provided by plugins.
+Any menu item can also be added to the context menu.
+To retrieve a list from the editor, run the following command from the browser console:
+
+```js
+tinymce.activeEditor.ui.registry.getAll().contextMenus
+```
+
+| identifier  | Core/Plugin  | Description  |
+| ----------- | ------------ | ------------ |
+| image | [Image]({{site.baseurl}}/plugins/image/) | Adds the _Image..._ item for opening the **Insert/Edit Image** dialog. |
```

Comment:
I just realised `linkchecker` and `configurepermanentpen` is missing in this list sorry @tylerkelly13! It's part of the default configuration, which is actually `'link linkchecker image imagetools table spellchecker configurepermanentpen'`

Reply:
I seem to recall we may have specifically left the `configurepermanentpen` out, but `linkchecker` should definitely be documented.

Reply:
ah, found out why these didn't come up for the full-featured demo:
```
 contextmenu: "link image imagetools table configurepermanentpen",
```
and i guess the permanent pen section predicate requires it to be enabled.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1495/commits/f92443739c56b95a243fa4984e7e662569ea787d